spec/index.html

```diff
@@ -1152,6 +1152,23 @@ <h3>The XML Schema Built-in Datatypes</h3>
         model.</li>
     </ul>
 
+    <p class="note">The <a>value spaces</a>
+      of <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a> and
+      <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a> do not
+      include all decimal numbers. For every literal of
+      any of these two datatypes, the value given as the <a>lexical form</a>
+      of the literal is rounded to a value that can be represented as an
+      <a href="http://ieeexplore.ieee.org/servlet/opac?punumber=4610933">IEEE 754-2008</a>
+      binary floating point representation of the corresponding precision.
+      As a consequence, not all decimal numbers are accurately captured by
+      using these two datatypes. For instance, the value of the literal with
+      lexical value `"0.1"` and datatype <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a>
+      is rounded to the floating point number 0.100000001490116119384765625.
```

Comment:
This might be read as it is just a issue of the representation in memory after parsing (as one would type `0.1f` in Java). I would put more emphasis on the deviating semantic of the literale itself.
```suggestion
      using these two datatypes. For instance, the literal with lexical value
      `"0.1"` and datatype <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a>
      denotes the number 0.100000001490116119384765625.
```

Reply:
The `<a href="#dfn-denote">` is actually incorrect; if it were external, you would use `data-cite` instead of `href`, however, as it's defined in RDF Concepts, it's sufficient to use `<a>denotes</a>`.

Reply:
That value has more digits than an xsd:float represents - putting in "floating point" is confusing as whether it is xsd:float or abstract point on the number line or is the xsd:double for the xsd:float.

Suggestion remove "floating point" and leaving as "the number".

Reply:
Putting the above suggestions together:

```suggestion
      using these two datatypes. For instance, the literal with
      lexical form `"0.1"` and datatype <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a>
      <a>denotes</a> the number 0.100000001490116119384765625.
```